doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -59,6 +59,62 @@ Protocols
 
 The issues in this section are related to :ref:`protocols`.
 
+Amazon Sidewalk
+===============
+
+.. rst-class:: v2-5-0
+
+KRKNWK-17860: QSPI assert occurs when performing DFU in the Sidewalk application mode
+  The DFU must be performed only in the DFU mode.
+  The DFU Bluetooth service can be used in the Sidewalk mode, however, using it leads to assertion failure, resulting in a Zephyr fatal error.
+
+.. rst-class:: v2-5-0
+
+KRKNWK-17800: After reconnecting to the network, the end device cannot find the route to its host
+  After the device disconnects from Sidewalk servers, the sensor monitoring app over Bluetooth LE shows an error ``-38 (SID_ERROR_NO_ROUTE_AVAILABLE)``.
+
+  **Workaround:** The device needs to be reset manually.
+
+.. rst-class:: v2-5-0
+
+KRKNWK-17750: Error occurs when sending multiple messages in a short period of time
+  When sending multiple Sidewalk messages in a short period of time, the internal queues might become full, showing misleading error messages, such as ``-12 (SID_ERROR_INCOMPATIBLE_PARAMS)``.
+
+  **Workaround:** The message must be resent after the protocol empties the queues.
+
+.. rst-class:: v2-5-0
+
+KRKNWK-17244: CMake warnings when building the Sidewalk samples
+  CMake warnings (``No SOURCES given to Zephyr library``) show up in the build log of a Sidewalk application.
+  The application builds successfully, but the error might obfuscate other important warnings.
+
+.. rst-class:: v2-5-0
+
+KRKNWK-17374: Sporadic Zephyr fatal error after disconnecting on FSK
+  After disconnecting on FSK, Zephyr fatal error occurs due to assertion in the semaphore module.
+  The error reproduces rarely (once per a few days).
+  The device resets automatically in the release mode, however, in the debug mode it needs to be reset manually.
+  Currently, this issue occurs for Sidewalk v1.14 libraries, and it will be fixed in a future version.
+
+.. rst-class:: v2-5-0
+
+KRKNWK-17035: Sensor monitor uplink messages are lost when the notification period is longer than 30 seconds
+  If the notification period is set to longer than 30 seconds, sensor monitor uplink messages are lost.
+
+  **Workaround:** The notification period is set to 15 seconds by default.
+
+.. rst-class:: v2-5-0
+
+KRKNWK-14583: Bus fault after flash, before the :file:`Nordic_MFG.hex` data flash
+  For sub-GHz samples, when the :file:`Nordic_MFG.hex` file is missing, the device throws a hard fault during initializing the Sidewalk stack.
+  Proper error handling will be implemented, but the temporary solution is to write a manufacturing hex to the device and reset it.
+
+.. rst-class:: v2-5-0
+
+KRKNWK-14299: NRPA MAC address cannot be set in Zephyr
+  The non-resolvable private address (NRPA) cannot be set in the connectable mode for Bluetooth LE.
+  Currently, there is no workaround for this issue.
+
 Bluetooth® LE
```

Comment:
There is 4 new known issus:

.. rst-class:: v2-5-0 v2-4-2 v2-4-1 v2-4-0 v2-3-0 v2-2-0 v2-1-4 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0 v1-8-0 v1-7-1

NCSDK-21625: Advertisements of Mesh GATT services are not stopped by :c:func:`bt_mesh_suspend` and not resumed by :c:func:`bt_mesh_resume
:c:func:`bt_mesh_suspend` and :c:func:`bt_mesh_resume does not work in conjunction with bt_disable() and bt_enable().

**Workaround:** To disable node identity advertisement use bt_mesh_subnet_node_id_set instead.

.. rst-class:: v2-5-0 v2-4-2 v2-4-1 v2-4-0 v2-3-0 v2-2-0 v2-1-4 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0 v1-8-0 v1-7-1

NCSDK-23087: Unsolicited Time Status rewrites periodic publishing TTL to zero forever
Time model specification mandates to publish unsolicited Time Status messages with ttl 0. The implementation rewrites the TTL field to zero, but does not write the initial value back then losing the initial value.

**Workaround:** Configure initial TTL value after an unsolicited Time Status is sent.

.. rst-class:: v2-5-0 v2-4-2 v2-4-1 v2-4-0 v2-3-0 v2-2-0 v2-1-4 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0 v1-8-0 v1-7-1

NCSDK-23220: The Heartbeat Publication Status message may be malformed after provisioning
After provisioning and obtaining Composition Data, reading the Heartbeat Publication and the Heartbeat Publication Status will contains garbage in NetKeyIndex field. The reason for this is that the field is not initially cleared.

.. rst-class:: v2-5-0

NCSDK-23308: Setting Storage causing device to reboot in case of clean operation
For non-secure builds, whenever a flash erase on setting storage happens, it causes device to reboot. The same issue can be seen where stored model data is big enough and changes that causes defragging/cleaning. EMDS partition is not effected by this.

Reply:
Updated. Please check.

Reply:
All of these should probably go under Bluetooth mesh section. Right @Balaklaka?

Reply:
Yes, they should be under Bluetooth mesh